pkg/apis/logging/v1/output_test.go

```diff
@@ -2,28 +2,67 @@ package v1_test
 
 import (
```

Comment:
it looks like the `pkg/apis/logging/v1/output.go` file has been deleted...

Reply:
No, the output stuff is now more cleanly split between output_types.go and the output/ sub-package. There wasn't anything left for output.go to do.

Reply:
Nope, the output code is now split between output_types,go and the output/ sub-package.

Reply:
my point i guess is more that `output_test.go` seems out of place.. should we rename this and create the `output_*_test.go` files as well?